internal/proto/data_coord.proto

```diff
@@ -486,6 +486,7 @@ message ImportTaskState {
 
 message ImportTaskInfo {
   int64 id = 1;                       // Task ID.
+  // TODO：remove request id cause it's useless
```

Comment:
Protobuf fields should not be removed, otherwise it might cause compatibility issue.

Instead, mark this field deprecated.

Reply:
Also remove all references of the field in the current PR.

Reply:
done